addons/stock/wizard/stock_quant_reserve.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_stock_quant_reserve_form" model="ir.ui.view">
+        <field name="name">Reserve stock</field>
+        <field name="model">stock.quant.reserve</field>
+        <field name="arch" type="xml">
+            <form>
+                <field name="move_id" invisible="1" />
+                <group>
+                    <group>
+                        <label for="qty_to_reserve" />
+                        <div class="o_row">
+                            <span><field name="qty_to_reserve" nolabal="1"/></span>
+                            <span><field name="product_uom_id" nolabal="1"/></span>
+                        </div>
+                    </group>
+                </group>
+                <div class="oe_grey">
+                    <p>Choose from the following available stock and enter the quantity to reserve.</p>
+                </div>
+                <field name="quant_line_ids" readonly="False" force_save="1">
```

Comment:
is there a reason why the readonly=False is in the view and not in the field declaration?